Week 2 Assignment 1/Factorial_W2A1_3.java

```diff
@@ -0,0 +1,15 @@
+public class Factorial_W2A1_3 {
+    static int product = 1;
```

Comment:
The "product" variable could have been local to the factorial function.  No need to share it between both functions.